website/blog/2024-04-15-0.74-post.md

```diff
@@ -0,0 +1,407 @@
+---
+title: 'React Native 0.74'
+authors:
+  [hurali, alanjhughes, alfonsocj, ingridwang, huntie, cortinico]
```

Comment:
```suggestion
  [hurali, alanjhughes, alfonsocj, huntie, cortinico]
```

Can we keep this limited to only the release crew @ingridwang ? Asking as otherwise we're going to have a lot of extra authors for everyone that writes a paragraph here

Reply:
Sure sounds good